src/test/java/searchengine/vsm/VectorSpaceModelTest.java

```diff
@@ -244,13 +245,13 @@ void singeTermSearchGermany() {
         scoredDocuments = vsm.getScoredDocuments( TFIDFDocuments );
         //assert
         assertEquals("test1", scoredDocuments.get(0).getTitle());
-        assertEquals(1.4606636323886684, scoredDocuments.get(0).getScore());
+        //assertEquals(2.1609640474436813, scoredDocuments.get(0).getScore());
 
         assertEquals("test4", scoredDocuments.get(1).getTitle());
-        assertEquals(1.356171465781643, scoredDocuments.get(1).getScore());
+        //assertEquals(2.1609640474436813, scoredDocuments.get(1).getScore());
 
-        assertEquals(0.0, scoredDocuments.get(2).getScore());
-        assertEquals(0.0, scoredDocuments.get(3).getScore());
+        //assertEquals(0.0, scoredDocuments.get(2).getScore());
+        //assertEquals(0.0, scoredDocuments.get(3).getScore());
```

Comment:
Gælder også her, generelt i hele filen :) 
Principielt bør vi også mocke databasen så vi ikke i tests snakker med DB, men lav prio for det er jo bare fixes til sidste års tests